__tests__/utils.tsx

```diff
@@ -173,11 +173,32 @@ describe('fetchWithCache()', () => {
     expect(await response.text()).toBe('test')
   })
 
+  test('responses are cached for 1 hour' , async () => {
+    server.use(
+      rest.get('http://example.com/', (_req, res, ctx) => {
+        return res.once(ctx.status(200), ctx.body('test'))
+      })
+    )
+    const fetchMock = jest.fn().mockResolvedValueOnce(new Response('http://example.com/'))
+
+    await fetchMock('http://example.com/')
+  })
+})
+
```

Comment:
This test doesn't make sense. Since you want to test `fetchWithCache` you need to call this function. You also need to have an actual test with `expect`. Revert this test to the original state and start rewriting it again.